application-services/custom/camera-management/README.md

```diff
@@ -22,54 +127,68 @@ make install-edge-video-analytics
 # Run the EVAM services (in another terminal)
 make run-edge-video-analytics
 # ...
-# Leave this running. If needed to stop
+# OPTIONAL: Leave this running. If you need to stop the services, run Ctrl-C
 make stop-edge-video-analytics
 ```
-3. Configure Camera Credentials
+
+### 3. Build and run the example application service
+
+#### 3.1 (Optional) Configure Onvif Camera Credentials.
+   > **Note:** This step is only required if you have Onvif cameras. Currently, this example app is limited to supporting
+   > only 1 username/password combination for all Onvif cameras.
 
    Option 1: Modify the `configuration.toml` file
+
    ```toml
    [Writable.InsecureSecrets.CameraCredentials]
    path = "CameraCredentials"
      [Writable.InsecureSecrets.CameraCredentials.Secrets]
-     username = ""
-     password = ""
+     username = "<username>"
+     password = "<password>"
    ```
-   
+
    Option 2: Export environment variable overrides
    ```shell
-   export WRITABLE_INSECURESECRETS_CAMERACREDENTIALS_SECRETS_USERNAME=<username>
-   export WRITABLE_INSECURESECRETS_CAMERACREDENTIALS_SECRETS_PASSWORD=<passowrd>
-   ```
+   export WRITABLE_INSECURESECRETS_CAMERACREDENTIALS_SECRETS_USERNAME="<username>"
+   export WRITABLE_INSECURESECRETS_CAMERACREDENTIALS_SECRETS_PASSWORD="<password>"
+   ```  
 
-4. Build and run the example application service. Web UI is used to view cameras, select models 
-   and start inference pipelines for camera video streams and also view inference results streams.
+#### 3.2 Build and run
 ```shell
 # Build the app. 
 make build-app
 
 # Run the app.
 make run-app
-# ...
-# Open your browser to http://localhost:59750
-# ...
-# Ctrl-C to stop it
 ```
 
-### Development and Testing of UI
+Open your browser to [http://localhost:59750](http://localhost:59750) to view the Web-UI.  
+The Web UI can be used to:
+- List 
+- Control PTZ cameras
+- Select inference models and start inference pipelines
+- View inference results from MQTT
+
+### Additional Development
+#### Development and Testing of UI
```

Comment:
We need to add a note here that these steps are only required for UI development or else it might confuse the users.